conf/supervisord-dev.conf

```diff
@@ -8,20 +8,23 @@ stdout_events_enabled=true
 stderr_events_enabled=true
 stopsignal = KILL
 stopasgroup = true
+autostart = %(ENV_ENABLE_WEB)s
```

Comment:
Added these to `supervisord-dev.conf` to enable testing of the new process kill switches. Previously these were only in `supervisord.conf` as they hadn't been needed in dev. They're technically still not really needed as you could use `make run-docker` to test the kill switches using `supervisord.conf`. But I figure may as well have support for these in `make dev` as well?